docs/30-quick-start.mdx

```diff
@@ -18,15 +18,15 @@ composer create-project symfony/skeleton rentals
 
 <Tabs>
     <TabItem value="Mac">
+    With MacOS, users will need to make sure to install pkgconf from homebrew as mentioned [here](https://formulae.brew.sh/formula/pkgconf) using the command 
+    
     ```shell
-    cd rentals
-    ## Compile the PHP code on MacOS using openssl
-    brew install openssl
-    export OPENSSL_DIR=$(brew --prefix openssl)
-    export LDFLAGS="-L$OPENSSL_DIR/lib"
-    export CPPFLAGS="-I$OPENSSL_DIR/include"
-    export PKG_CONFIG_PATH="$OPENSSL_DIR/lib/pkgconfig"
+    brew install pkgconf
```

Comment:
Can we trust that `openssl@3` will already be installed in Homebrew? Just asking as that's what we're expecting to find with `pkg-config`.

Reply:
Homebrew includes an alias for `openssl` that points to `openssl@3`, so unless people explicitly install `openssl@1` they'll get the expected version.

I am a bit curious about "expecting to find" here - we're looking for openssl >= 1.0.1 in CheckSSL.m4, so I'm not sure why we're expecting to find openssl 3.

Reply:
Apologies. I just saw my local package and forgot that `CheckSSL.m4` only requires 1.0.1+ when I wrote that.

It was less about suggesting version 3+ and more about whether we should tell users they need `openssl` itself installed. I don't recall installing it manually in Homebrew. I think it was just pulled in by some other packages.

Reply:
Yes, we may consider suggesting people install `openssl` as well - even though there is a high likelihood that it will have been installed as a dependency for other development tools.